### PR TITLE
{bio} [Celera Assembler aka Whole-Genome Shotgun Assembler ] (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/457fe7c03149809bd0e9 for a full test report.

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+files_to_copy = ["Linux-amd64/*"]
+        
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["bogus", "convert-fasta-to-v2.pl", "fastqAnalyze", "markUniqueUnique",
+                                     "pacBioToCA", "uidclient"]],
+    'dirs': ["lib"],
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb]: @pescobar: remove empty line and address other comments

**Comment by pescobar** [comment]
@pforai I have updated this easyconfig because based on feedback from my users it requires to keep the hardcoded path "Linux-amd64"

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+source_urls = [('http://sourceforge.net/projects/wgs-assembler/files/wgs-assembler/wgs-%(version)s/', 'download')]
+sources = ['wgs-%(version)s.tar.bz2']
+
+parallel = 1
+
+prebuildopts = 'cd kmer && make install && cd .. && '
+prebuildopts += 'cd samtools && make && cd .. && '
+prebuildopts += 'cd src && '
+
+files_to_copy = ["*"]
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb]: so, use `ConfigureMake` with `skipsteps = ['install']` and `buildininstalldir = True`?

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.2-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+source_urls = [('http://sourceforge.net/projects/wgs-assembler/files/wgs-assembler/wgs-%(version)s/', 'download')]
+sources = ['wgs-%(version)s.tar.bz2']
+
+parallel = 1
+
+prebuildopts = 'cd kmer && make install && cd .. && '
+prebuildopts += 'cd samtools && make && cd .. && '
+prebuildopts += 'cd src && '
+
+files_to_copy = ["*"]
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.2-goolf-1.4.10.eb]: `buildininstalldir`?

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.2-goolf-1.4.10.eb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+sanity_check_paths = {
+    'files': ["Linux-amd64/bin/runCA"],
+    'dirs': [],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.2-goolf-1.4.10.eb]: also list `Linux-amd64/lib`?

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.3rc2-goolf-1.4.10-Python-2.7.5.eb [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+prebuildopts = 'cd kmer && make install && cd .. && '
+prebuildopts += 'cd src && '
+
+files_to_copy = ["*"]
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.3rc2-goolf-1.4.10-Python-2.7.5.eb]: buildininstalldir

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.3rc2-goolf-1.4.10-Python-2.7.5.eb [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+sanity_check_paths = {
+    'files': ["Linux-amd64/bin/runCA"],
+    'dirs': [],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.3rc2-goolf-1.4.10-Python-2.7.5.eb]: also list Linux-amd64/lib?

File: easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+sanity_check_paths = {
+    'files': ["Linux-amd64/bin/runCA"],
+    'dirs': [],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb]: also list Linux-amd64/lib?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2518151637251d076abd for a full test report.

**Comment by boegel** [comment]
@pescobar: ping on remarks?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**